waspc/examples/todoApp/main.wasp

```diff
@@ -39,10 +39,10 @@ app todoApp {
         },
       },
     },
-    hooks: {
-      postSignup: import { postSingupHook } from "@src/auth/hooks.js",
-      postLogin: import { postLoginHook } from "@src/auth/hooks.js",
-    },
+    // hooks: {
```

Comment:
What if we don't group them under `hooks`? We don't do it for other hooks in Wasp. Also, if they are using only one or two hooks, which is likely, they don't get much with `hooks` bring in front.